utils/oc.py

```diff
@@ -128,6 +129,7 @@ def get_items(self, kind, **kwargs):
 
         return items
 
+    @elapsed_seconds_from_commit_metric
```

Comment:
I don't think we need to track this ATM, only calls that write into k8s

Reply:
sorry, leftover from testing :]

Reply:
thx!